Common++/src/OUILookup.cpp

```diff
@@ -8,103 +8,104 @@
 namespace pcpp
 {
 
-template <typename T>
-int64_t OUILookup::internalParser(T &jsonData)
-{
-	// Clear all entries before adding
-	vendorMap.clear();
-
-	int64_t ctrRead = 0;
-	nlohmann::json parsedJson = nlohmann::json::parse(jsonData);
-	for (const auto &line : parsedJson.items())
+	template <typename T> int64_t OUILookup::internalParser(T& jsonData)
 	{
-		if (!(line.value().is_object()))
-			continue;
-		auto val = line.value().get<nlohmann::json>();
-		if (!(val.contains("vendor")))
-			continue;
-
-		std::vector<MaskedFilter> vLocalMaskedFilter;
-		if (val.contains("maskedFilters") && val["maskedFilters"].is_array())
+		// Clear all entries before adding
+		vendorMap.clear();
+
+		int64_t ctrRead = 0;
+		nlohmann::json parsedJson = nlohmann::json::parse(jsonData);
+		for (const auto& line : parsedJson.items())
 		{
-			// Iterate through masked filters
-			for (const auto &entry : val["maskedFilters"])
+			if (!(line.value().is_object()))
+				continue;
+			auto val = line.value().get<nlohmann::json>();
+			if (!(val.contains("vendor")))
+				continue;
+
+			std::vector<MaskedFilter> vLocalMaskedFilter;
+			if (val.contains("maskedFilters") && val["maskedFilters"].is_array())
 			{
-				if (!entry.is_object())
-					continue;
-				auto subVal = entry.get<nlohmann::json>();
-				if (subVal.contains("mask") && subVal.contains("vendors") && subVal["mask"].is_number_integer() &&
-					subVal["vendors"].is_object())
+				// Iterate through masked filters
+				for (const auto& entry : val["maskedFilters"])
 				{
-					int maskValue = subVal["mask"].get<int>();
-					vLocalMaskedFilter.push_back({maskValue, {}});
-
-					// Parse masked filter
-					for (const auto &subentry : subVal["vendors"].items())
+					if (!entry.is_object())
+						continue;
+					auto subVal = entry.get<nlohmann::json>();
+					if (subVal.contains("mask") && subVal.contains("vendors") && subVal["mask"].is_number_integer() &&
+					    subVal["vendors"].is_object())
 					{
-						if (subentry.value().is_string())
+						int maskValue = subVal["mask"].get<int>();
+						vLocalMaskedFilter.push_back({ maskValue, {} });
+
+						// Parse masked filter
+						for (const auto& subentry : subVal["vendors"].items())
 						{
-							vLocalMaskedFilter.back().vendorMap.insert(
-								{std::stoull(subentry.key()), subentry.value()});
-							++ctrRead;
+							if (subentry.value().is_string())
+							{
+								vLocalMaskedFilter.back().vendorMap.insert(
+								    { std::stoull(subentry.key()), subentry.value() });
+								++ctrRead;
+							}
 						}
 					}
 				}
 			}
+
+			vendorMap.insert({
+			    std::stoull(line.key()), { val["vendor"], vLocalMaskedFilter }
+            });
```

Comment:
There is a weird mix of tabs and spaces in these lines, but not sure if it can be fixed 🤔

Reply:
it is in purpose. the indent is tab, the alignment is space.
for example:
```
<tab> void  foo(int a,
<tab><spaces>int b) {}
```

Reply:
yeah, but why does line 57 has spaces instead of tabs? Maybe we should just move `});` to line 56?

Reply:
yeah... you are right.. it's weird here as well.

Reply:
did you try moving `});` from line 57 to line 56 before running `clang-format`?

Reply:
I tried, it doesn't help 
no matter this;
```
vendorMap.insert({std::stoull(line.key()), { val["vendor"], vLocalMaskedFilter }});
```
or
```
			vendorMap.insert(
				{std::stoull(line.key()), { val["vendor"], vLocalMaskedFilter }});
```

they will be formatted as:
```
			vendorMap.insert({
				std::stoull(line.key()), { val["vendor"], vLocalMaskedFilter }
            });
```

one way is to use `// clang-format off` here, but I would suggest just let it be this if it doesn't break the build.
according to my experience, when there is too many braces, clang-format becomes stupid.

Reply:
yes, we can just leave it the way it is... 😕